src/run.py

```diff
@@ -73,7 +74,7 @@ def export(self, export_args: Dict) -> None:
             exporter = dataset2exporter[dataset](self.data)
 
             for variable in variables:
-                _ = exporter.export(**variable)
+                _ = exporter.export(**variable)  # type: ignore
 
     def run(self, config: DictWithDefaults) -> None:
```

Comment:
We should add a processing function here so that it can be run as part of the pipeline.

It should also be added to the configuration files

Reply:
Done!